queenbee/recipe/dag.py

```diff
@@ -442,22 +65,22 @@ class DAGTask(BaseModel):
         'wanting to save results in a specific folder.'
     )
 
-    outputs: DAGTaskOutputs = Field(
+    returns: List[TaskReturns] = Field(
```

Comment:
This will also make things less confusing hopefully 😀

Reply:
I hope so. Since it is a breaking change. 😵